### ISSUE TITLE
Trouble Running Examples

### ISSUE THREAD

**Issue by cai-michael** [emmijokinen/TCRGP#1]
Hi,

I think in your recent update to tcrgp.py you started passing in a parameter "l" when calling construct_rbf_kernel.

https://github.com/emmijokinen/TCRGP/blob/d9dce045089fe3859473e670202eb79a4067fd2c/tcrgp.py#L821

I tried to run the Examples.ipynb file sequentially but I get stuck on the "Training, saving and loading models" cell because l is not defined. I think it might have just been a small oversight considering you define another variable l in the construct_rbf_kernel method itself and the corresponding variable in the signature has a default value already.

```
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-44-540c393ea8df> in <module>
      1 auc, params = tcrgp.train_classifier('training_data/examples/vdj_human_ATDALMTGY.csv','human','ATDALMTGY',pc_blo,
      2                                      cdr_types=[[],['cdr3']],m_iters=5000,lr=0.005,nZ=0,mbs=0,lmax3=None,
----> 3                                      va='va',vb='vb',cdr3a='cdr3a',cdr3b='cdr3b',epis='epitope')

~/Desktop/Codebases/TCRGP/tcrgp.py in train_classifier(datafile, organism, epi, pc, cdr_types, m_iters, lr, nZ, mbs, lmax3, clip, delimiter, va, vb, cdr3a, cdr3b, epis, check_v, balance_controls)
    965 
    966     with tf.Session(graph=tf.Graph()):
--> 967         kernel = construct_rbf_kernel(d,lmaxes,l,var)
    968         if nZ == 0:
    969             m = gpflow.models.VGP(X,y,kernel,gpflow.likelihoods.Bernoulli())

NameError: name 'l' is not defined
```

**Comment by XYZuo** [comment]
Hi cai-michael,
I met the same problem.How did you fix with that?
Thanks a lot.

**Comment by cai-michael** [comment]
Sorry for not getting back to you @ZxyChopcat, it's been awhile since I looked at this but I think I fixed this by adding `l=[1.0]` before that line.
It seems to be declared as a default parameter in several other methods but I think it was forgotten when making the signature for `train_classifier`

**Comment by emmijokinen** [comment]
Hi,
Thank you for pointing this out! The default initial values for the lengthscale and the kernel variance (l and var) should have been determined in the same way as with e.g. loo and loso functions. I have now updated this. cai-michael's fix works as well as long as you don't want use different initial values for the lengthscale and variance parameters.
